tests/test_core.py

```diff
@@ -378,19 +378,19 @@ def test_asmjs_unknown_emscripten(self):
   def test_cube2md5(self):
     self.emcc_args += ['--embed-file', 'cube2md5.txt']
     shutil.copyfile(path_from_root('tests', 'cube2md5.txt'), 'cube2md5.txt')
-    self.do_run(open(path_from_root('tests', 'cube2md5.cpp')).read(), open(path_from_root('tests', 'cube2md5.ok')).read())
+    self.do_run(open(path_from_root('tests', 'cube2md5.cpp')).read(), open(path_from_root('tests', 'cube2md5.ok')).read(), assert_returncode=None)
```

Comment:
Why is this expected to fail?

Reply:
I didn't investigate each test in depth, I just saw which currently only pass when accepting failure and marked them as such. That is enough to switch the default, which improves the current status quo for 99% of tests, and for new tests.

Reading this test, it looks like it returns 1 from main.

It might be cleaner to go test by test and investigate/fix, but I wonder if it's worth the time?